src/main/java/net/sf/rails/game/financial/ShareSellingRound.java

```diff
@@ -164,99 +175,136 @@ protected void findSellableShares() {
             int dumpThreshold = 0;
             SortedSet<Integer> possibleSharesToSell = null;
             boolean dumpIsPossible = false;
+            Player dumpedPlayer = company.findPlayerToDump();
+            int dumpedPlayerShares = dumpedPlayer != null
```

Comment:
Hm ? you are assigned a Player dumpedPlayer to an Int value dumpedPlayerShares and the compiler is not complaining ? Or what am i missing here ?

Reply:
You mean this bit?
`            int dumpedPlayerShares = dumpedPlayer != null
                    ? dumpedPlayer.getPortfolioModel().getShares(company)
                    : 0;
`
That's a ?: type if/then/else. Read it as
`            int dumpedPlayerShares = (dumpedPlayer != null
                    ? dumpedPlayer.getPortfolioModel().getShares(company)
                    : 0);
`

Reply:
Missing the parantheses there

Reply:
GitHub was misleading you by cutting off in the middle of that statement.

Reply:
PR updated to add 1835 Bankruptcy handling, improbable as that may be.

Oh, and a pair of parentheses on behalf of Martin.